### PR TITLE
Adds DROPDEL flag; items will qdel() on dropped()

### PR DESCRIPTION
**Pull request by coiax**
Pretty self explanatory, replaces writing a dropped() proc that just
calls qdel() with a flag that does it for you.
- Also replaces a spawn() in energy cuffs with an addtimer

### CODE REVIEW

File: code/game/objects/items/weapons/implants/implant.dm [hunk 1]
```diff
@@ -73,12 +74,9 @@
 		removed(imp_in)
 	return ..()
 
-
 /obj/item/weapon/implant/proc/get_data()
 	return "No information available"
 
 /obj/item/weapon/implant/dropped(mob/user)
 	..()
 	. = 1
```
Comment (phil235) [inline code/game/objects/items/weapons/implants/implant.dm]: since qdel() is called in the ..() shouldn't it be safer to have . = 1 before the ..() ?

Comment (coiax) [inline code/game/objects/items/weapons/implants/implant.dm]: I have no idea what the . = 1 is for in this case.

But sure.

**Comment by hornygranny** [comment]
good idea

**Comment by optimumtact** [comment]
Good idea

**Pull request merged**